webapp/src/js/bootstrapper/initial-replication.js

```diff
@@ -50,40 +48,15 @@ const getDataUsage = () => {
   }
 };
 
-const displayTooManyDocsWarning = ({ warn_docs, limit }) => {
-  return new Promise(resolve => {
-    const translateParams = { count: warn_docs, limit: limit };
-    const errorMessage = translator.translate('TOO_MANY_DOCS', translateParams);
-    const continueBtn = translator.translate('CONTINUE');
-    const abort = translator.translate('ABORT');
-    const content = `
-            <div>
-              <p class="alert alert-warning">${errorMessage}</p>
-              <a id="btn-continue" class="btn btn-primary pull-left" href="#">${continueBtn}</a>
-              <a id="btn-abort" class="btn btn-danger pull-right" href="#">${abort}</a>
-            </div>`;
-
-    $('.bootstrap-layer .loader, .bootstrap-layer .status').hide();
-    $('.bootstrap-layer .error').show();
-    $('.bootstrap-layer .error').html(content);
-    $('#btn-continue').click(() => resolve());
-    $('#btn-abort').click(() => {
-      document.cookie = 'login=force;path=/';
-      window.location.reload(false);
-    });
-  });
-};
-
 const getMissingDocIdsRevsPairs = async (localDb, remoteDocIdsRevs) => {
   const localDocs = await getLocalDocList(localDb);
   return remoteDocIdsRevs.filter(({ id, rev }) => !localDocs[id] || localDocs[id] !== rev);
 };
 
-const getDownloadList = async (localDb) => {
+const getDownloadList = async (localDb = true) => {
   const response = await utils.fetchJSON('/api/v1/initial-replication/get-ids');
```

Comment:
This is the old URL, right? I take it there's some more work to do here to migrate over. There also might be enough common code to reuse between here and the replication service.